lib/controller/firebase_controller.dart

```diff
@@ -576,10 +576,24 @@ class FirebaseController {
   Future<void> removeCollection({
     required BookCollection collection,
     required List<Item> items,
-  }) {
+  }) async {
     try {
-      // TODO(@Suman1415): Implement removeCollection method
-      throw UnimplementedError();
+      for (final item in items) {
+        //remove collection id. Dart implements remove function inplace
+        item.collectionIds.remove(collection.id);
```

Comment:
You should not modify the original item to avoid sync issues in case the firebase call fails. Instead, copy the list and remove the collection id from the copy.
```dart
final List<Item> collectionIds = [...item.collectionIds.toList()..remove(collection.id)];
```
and pass in the new `collectionIds` variable to the update collections functions below